crates/krun/src/bin/krun.rs

```diff
@@ -1,6 +1,7 @@
 use std::{
     env::{self, VarError},
     ffi::{c_char, CString},
+    fs::read_to_string,
```

Comment:
minor style nit:

Only import `fs` here, and do `fs::read_to_string` at the call site below.

Same as the style in the example:
https://doc.rust-lang.org/stable/std/fs/fn.read_to_string.html

Reply:
(This is mainly because there are both `fs::read_to_string` and `io::read_to_string` in std... So it would be ambiguous to the reader at a glance.)